Firestore/core/src/remote/existence_filter.h

```diff
@@ -18,36 +18,52 @@
 #define FIRESTORE_CORE_SRC_REMOTE_EXISTENCE_FILTER_H_
 
 #include <utility>
+#include <vector>
 
 #include "Firestore/core/src/remote/bloom_filter.h"
 
 namespace firebase {
 namespace firestore {
 namespace remote {
 
+struct BloomFilterParameter {
```

Comment:
nit: Rename `BloomFilterParameter` to `BloomFilterParameters` (i.e. make it plural) because it contains 3 parameters.